plugins/clipboard-manager/guest-js/index.ts

```diff
@@ -49,12 +54,65 @@ async function writeText(
  * @since 2.0.0
  */
 async function readText(): Promise<string> {
-  const kind: ClipResponse = await invoke("plugin:clipboard-manager|read");
+  const kind: ClipResponse = await invoke("plugin:clipboard-manager|read_text");
   return kind.plainText.text;
 }
 
 /**
- * Writes HTML or fallbacks to write provided plain text to the clipboard.
+ * Gets the clipboard content as Uint8Array image.
+ * @example
+ * ```typescript
+ * import { readImage } from '@tauri-apps/plugin-clipboard-manager';
+ *
+ * const clipboardImage = await readImage();
+ * const blob = new Blob([clipboardImage.bytes], { type: 'image' })
+ * const url = URL.createObjectURL(blob)
+ * ```
+ * @since 2.0.0
+ */
+async function readImage(): Promise<ClipboardImage<Uint8Array>> {
```

Comment:
this should return an instance of `Image` from `@tauri-apps/image`

Reply:
in this case we already have the bytes/width/height and the Image class does note expose an API to directly create an image from those values.. doesn't make much sense to send them back to the rust side...

Reply:
The `Image` class should be constructed on the Rust side then added to the resources_table, we only return an RresourceId to JS and construct the Js Image class with it.